.drone.star

```diff
@@ -108,6 +128,35 @@ def build(name, arch, tag, publish):
 
     return step
 
+def manifest(name, platforms, tag):
+    imageRepo = "public.ecr.aws/kanopy/buildah-plugin"
```

Comment:
this is bad since it's not sourcing the registry from drone secrets, but I don't know if there's a way to do that.

Reply:
This is a public registry on a public repo,